framework/test/unit/modules/auth/api.spec.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { AuthAPI } from '../../../../src/modules/auth/api';
+import { STORE_PREFIX_AUTH } from '../../../../src/modules/auth/constants';
+import { authAccountSchema } from '../../../../src/modules/auth/schemas';
+import { createTransientAPIContext } from '../../../../src/testing';
+
+describe('AuthAPI', () => {
+	let authAPI: AuthAPI;
+	let context: any;
+	let authStore: any;
+	const address = Buffer.from('fa1c00809ff1b10cd269a711eef40a465ba4a9cb', 'hex');
+
+	beforeEach(async () => {
+		authAPI = new AuthAPI(1);
+		context = createTransientAPIContext({});
+		authStore = context.getStore(authAPI['moduleID'], STORE_PREFIX_AUTH);
+		await authStore.setWithSchema(
+			address,
+			{
+				nonce: 1,
+				numberOfSignatures: 1,
+				mandatoryKeys: [],
```

Comment:
```suggestion
				mandatoryKeys: [getRandomBytes(64), getRandomBytes(64)],
```
I think it would be better to have something to make sure it is properly obtained. same for `optionalKeys`